cdapython/utils/utility.py

```diff
@@ -17,7 +17,7 @@
 from cdapython.exceptions.custom_exception import HTTP_ERROR_API, HTTP_ERROR_SERVICE
 from cdapython.results.columns_result import ColumnsResult
 from cdapython.results.factories.collect_result import CollectResult
-from cdapython.results.page_result import get_query_result
+from cdapython.results.page_result import Paged_Result, get_query_result
```

Comment:
a nit pick - usually I would expect to see either PagedResult or paged_result. This seems like a strange mix of the two. was there a specific reason to do this?